go-controller/pkg/allocator/pod/pod_annotation.go

```diff
@@ -456,3 +457,27 @@ func allocatePodAnnotationWithRollback(
 
 	return
 }
+
+func shouldSkipAllocateIPsError(err error, networkAllocated bool, ipamClaim *ipamclaimsapi.IPAMClaim) bool {
```

Comment:
could we avoid passing the boolean attribute ? you can check that in the caller.

Reply:
I disagree, but I'm not a maintainer. 
Whatever they say.